### PR TITLE
[SDK-871] Add React bindings for the Viewer SDK

### PR DESCRIPTION
**Pull request by danschultz**
## What

Adds React bindings for the Viewer SDK to make it easier to use our SDK in a React application.

```jsx
import React from 'react';
import { VertexViewer } from '@vertexvis/viewer-react';

export function App() {
  return (<div>
    <VertexViewer src="urn:vertexvis:stream-key:123" />
  </div>);
}
```

You'll need to use the provided loader before the components will be loaded and rendered.

```jsx
import React from 'react';
import ReactDom from 'react-dom';
import { defineCustomElements } from '@vertexvis/viewer-react';

function main() {
  defineCustomElement(window);
  ReactDom.render(<App />, document.querySelector("#app"));
}

main();
```

## Ticket

https://vertexvis.atlassian.net/browse/SDK-871

## Test Plan

Tested loading the component in a React application compiled with Webpack.

## Areas of Possible Regression

N/A

## Out of scope changes made in PR

N/A

### CODE REVIEW

File: packages/viewer-react/package.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    "test": "echo \"Error: run tests from root\" && exit 1"
+  },
+  "dependencies": {
+    "@vertexvis/viewer": "^0.6.0"
+  },
+  "devDependencies": {
+    "@types/react": "^16.9.46",
+    "@types/react-dom": "^16.9.8",
+    "@vertexvis/eslint-config-vertexvis-typescript": "0.2.2",
+    "@vertexwebsdk/build": "^0.6.0",
```
Comment (jdm717) [inline packages/viewer-react/package.json:40]: Curious, where is this dependency coming from? I can't seem to find it on NPM, but I'm able to install/build the branch just fine

Comment (danschultz) [reply packages/viewer-react/package.json:40]: It's from a package in the `./internal/build` folder.

File: packages/viewer-react/package.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  "author": "Dan Schultz <dan.schultz@vertexvis.com>",
+  "homepage": "https://github.com/Vertexvis/vertex-web-sdk#readme",
+  "bugs": {
+    "url": "https://github.com/Vertexvis/vertex-web-sdk/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Vertexvis/vertex-web-sdk.git"
+  },
+  "license": "ISC",
```
Comment (danschultz) [inline packages/viewer-react/package.json:14]: Need to update.

File: packages/viewer-react/package.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    "registry": "https://registry.npmjs.org",
+    "access": "public"
+  },
+  "scripts": {
+    "clean": "rm -fr ./dist && mkdir ./dist",
+    "prebuild": "yarn clean",
+    "build": "rollup --config ./rollup.config.js",
+    "format": "yarn lint --fix",
+    "lint": "eslint --ext .ts,.tsx,.js,.jsx --ignore-path ../../.gitignore --ignore-pattern 'src/generated' .",
+    "test": "echo \"Error: run tests from root\" && exit 1"
```
Comment (danschultz) [inline packages/viewer-react/package.json:31]: Need to update.

**Pull request merged**